content/reference-transaction-format.md

```diff
@@ -1365,88 +1365,89 @@ These codes indicate that the transaction failed and was not included in a ledge
+| `tecNO_AUTH`                 | 134   | The transaction failed because it needs to add a balance on a trust line to an account with the `lsfRequireAuth` flag enabled, and that trust line has not been authorized. If the trust line does not exist at all, `tecNO_LINE` occurs instead. |
+| `tecNO_DST`                  | 124   | The account on the receiving end of the transaction does not exist. This includes Payment and TrustSet transaction types. (It could be created if it received enough XRP.) |
+| `tecNO_DST_INSUF_XRP`       | 125   | The account on the receiving end of the transaction does not exist, and the transaction is not sending enough XRP to create it. |
+| `tecNO_ENTRY`                | 140   | Reserved for future use.               |
+| `tecNO_ISSUER`               | 133   | The account specified in the `issuer` field of a currency amount does not exist. |
+| `tecNO_LINE`                 | 135   | The `TakerPays` field of the [OfferCreate transaction][] specifies an asset whose issuer has `lsfRequireAuth` enabled, and the account making the offer does not have a trust line for that asset. (Normally, making an offer implicitly creates a trust line if necessary, but in this case it does not bother because you cannot hold the asset without authorization.) If the trust line exists, but is not authorized, `tecNO_AUTH` occurs instead. |
+| `tecNO_LINE_INSUF_RESERVE` | 126   | The transaction failed because the sending account does not have enough XRP to create a new trust line. (See: [Reserves](concept-reserves.html)) This error occurs when the counterparty does not have a trust line to this account for the same currency. (See `tecINSUF_RESERVE_LINE` for the other case.) |
+| `tecNO_LINE_REDUNDANT`      | 127   | The transaction failed because it tried to set a trust line to its default state, but the trust line did not exist. |
+| `tecNO_PERMISSION`           | 139   | The sender does not have permission to perform this operation. For example, the [EscrowFinish transaction][] tried to release a held payment before its `FinishAfter` time, or someone tried to use [PaymentChannelFund][] on a channel the sender does not own. |
+| `tecNO_REGULAR_KEY`         | 131   | The [AccountSet transaction][] tried to disable the master key, but the account does not have another way to [authorize transactions](#authorizing-transactions). If [multi-signing](#multi-signing) is enabled, this code is deprecated and `tecNO_ALTERNATIVE_KEY` is used instead. |
+| `tecNO_TARGET`               | 138   | The destination account does not exist, or it does exist but has `asfDisallowXRP` set so it cannot be the destination of this [PaymentChannelCreate][] or [EscrowCreate][] transaction. |
```

Comment:
`tecNO_TARGET` is returned if a transaction specifies an payment channel or escrow whose ledger entry does not exist (either it never existed or already closed).
It indicates an invalid (not found) `Channel` for `PaymentChannelFund` and `PaymentChannelClaim`.
It indicates an invalid `Owner` + `OfferSequence` for `EscrowCancel` and `EscrowFinish`.
This is incorrect:
>The destination account does not exist

`tecNO_DST` would be returned in that situation
It sounds like we change it so that `tecNO_TARGET` is not returned in the `asfDisallowXRP` case.